catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/util/UiTestUtils.java

```diff
@@ -89,4 +102,18 @@ public static boolean comparePixelRgbaArrays(byte[] firstArray, byte[] secondArr
 		}
 		return true;
 	}
+
+	public static void addBrick(Class<?> brickHeaderClass, int brickCategoryId) {
```

Comment:
this has no business in the global UiTestUtils function, this function can only work in script fragment, and is only used in brick tests, so please move it to the utility function in that test package